src/js/post-911-gib-status/components/EnrollmentHistory.jsx

```diff
@@ -22,28 +21,38 @@ class EnrollmentHistory extends React.Component {
     */
 
     // History explanation box
+    let historyExplanationBox;
     const benefitMaybePending =
-      (enrollmentData.percentageBenefit > 0 || enrollmentData.originalEntitlement > 0) &&
-      enrollmentData.usedEntitlement === 0;
+      (enrollmentData.percentageBenefit > 0 || enrollmentData.originalEntitlement.months > 0) &&
+      (enrollmentData.usedEntitlement.months === 0 && enrollmentData.usedEntitlement.days === 0);
     const historyMayLookIncorrect = enrollments.length > 0 || benefitMaybePending;
-    const noEnrollmentHistory = enrollments.length === 0 && enrollmentData.usedEntitlement > 0;
-    const historyHeading = noEnrollmentHistory ?
-                           (<h4>You don't have any enrollment history</h4>) :
-                           (<h4>Does something look wrong in your enrollment history?</h4>);
-    const historyExplanation = noEnrollmentHistory ?
-                               (<span>Your enrollment history may not be available if:</span>) :
-                               (<span>Certain enrollments may not be displayed in this history if:</span>);
-    const historyExplanationBox = (historyMayLookIncorrect || noEnrollmentHistory) && (
-      <div className="feature">
-        {historyHeading}
-        {historyExplanation}
-        <ul>
-          <li>Your school made a request to us that's still in process, or</li>
-          <li>You made a request to us that's still in process, or</li>
-          <li>You used or are using your benefit for flight, on-the-job, apprenticeship, or correspondence training</li>
-        </ul>
-      </div>
-    );
+    const noEnrollmentHistory = enrollmentData.usedEntitlement.months === 0 && enrollmentData.usedEntitlement.days === 0;
```

Comment:
Looking at https://marvelapp.com/8d6igd9/screen/29281166, it doesn't look like the 'no enrollment history' case is conditioned on the months/days of usedEntitlement. I actually don't see this case listed in the scenarios in https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3839 (guessing that's because we implemented it a while back). Do you see in some ticket that the condition that was here before (`enrollments.length === 0 && enrollmentData.usedEntitlement > 0`) is no longer the right one?